console-extensions.json

```diff
@@ -115,21 +115,10 @@
     "type": "console.navigation/section",
     "properties": {
       "id": "cryostat-section",
-      "insertAfter": "home",
       "perspective": "admin",
       "name": "%plugin__cryostat-plugin~Cryostat%"
     }
   },
-  {
-    "type": "console.navigation/href",
-    "properties": {
-      "id": "example",
-      "name": "%plugin__cryostat-plugin~Navigation.Plugin_Example%",
-      "href": "/cryostat/example",
-      "perspective": "admin",
-      "section": "cryostat-section"
-    }
-  },
```

Comment:
I think at this point we can just go ahead and delete whole whole ExamplePage component, exposed module, etc.